src/integration_tests/schemas/e2e_schema_integration_test.py

```diff
@@ -59,6 +59,14 @@ def test_schema_e2e(self):
         response_as_json = test_schema_get_response.json()
         assert len(response_as_json) > 0
 
+        set_list_survey_id_response = session.get(
+            f"{config.API_URL}/v1/survey_list",
+            headers=headers,
+        )
+
+        assert set_list_survey_id_response.status_code == 200
+        assert set_list_survey_id_response.json() == ["test_survey_id"]
```

Comment:
This test seems not quite sufficient

Reply:
@IsaiahLai111 Well spotted, just added more tests to add more schemas with the same survey id multiple times and then check that the new endpoint returns only the unique values, hope this is sufficient now, cheers.